packages/flutter/test/material/chip_theme_test.dart

```diff
@@ -161,11 +174,54 @@ void main() {
     final RenderBox materialBox = getMaterialBox(tester);
     expect(materialBox, paints..rect(color: chipTheme.backgroundColor));
     expect(getMaterial(tester).elevation, chipTheme.elevation);
-    expect(tester.getSize(find.byType(RawChip)), const Size(250, 250)); // label + padding + labelPadding
+    expect(tester.getSize(find.byType(RawChip)), const Size(400, 250)); // icon + label + padding + labelPadding
     expect(getMaterial(tester).shape, chipTheme.shape);
     expect(getLabelStyle(tester).style.fontSize, 32);
+    expect(tester.widget<IconTheme>(find.widgetWithIcon(IconTheme,Icons.ac_unit).first).data.color, customIconTheme.color);
+    expect(getIconTheme(tester).color, customIconTheme.color);
```

Comment:
what's the difference b/w these two? aren't they just two different ways to find the same `IconTheme` widget? the second one will suffice.